packages/devtools_app/lib/src/profiler/cpu_profile_service.dart

```diff
@@ -30,7 +30,7 @@ class CpuProfilerService {
   ValueNotifier<Flag> get profileGranularityFlagNotifier =>
       serviceManager.vmFlagManager.flag(vm_flags.profilePeriod);
 
-  Future<Success> clearCpuSamples() {
+  Future<dynamic> clearCpuSamples() {
```

Comment:
For cases like this where we don't really care about the return value default to just returning
Future<void>

Reply:
done